src/__tests__/graphqlObservable-test.ts

```diff
@@ -396,6 +400,22 @@ describe("graphqlObservable", function() {
           m.expect(result.pipe(take(1))).toBeObservable(expected);
         });
 
+        // fixme: without `only` all tests pass 🤔
+        itMarbles.only("if defined but returns undefined, field is null", function (m) {
```

Comment:
I sense async induced race conditions. Will check

Reply:
It's actually because of this: https://github.com/mesosphere/reactive-graphql/blob/master/src/__tests__/graphqlObservable-test.ts#L338

28 tests skipped with this, 8 without

Reply:
After merging #24, this kind of fix might be required: https://github.com/getstation/reactive-graphql/commit/1034b763621942bc3e4609587c68802f055be912

Reply:
Done in aafb8e0